doc/release_notes_0.2.rst

```diff
@@ -17,8 +17,63 @@ ACRN device model, and documentation.
 Version 0.2 new features
 ************************
 
+VT-x, VT-d
+================
+Based on Intel VT-x virtualization technology, ACRN emulates virtual CPU with core partition and simple schedule.
+VT-d stands for Intel VT for Directed IO, providing capabilities to assign I/O devices to VMs and extending the protection and isolation properties of VMs for I/O operations.
+
+PIC/IOAPIC/MSI/MSI-X/PCI/LAPIC
+================
+ACRN hypervisor supported APIC-V/EPT/IOAPIC/LAPIC functionality.
+
+Ethernet
+================
+ACRN hypervisor supported Ethernet functionality. Ethernet Mediator is executed in the Service OS and  provided packet forwarding between the physical networking devices (Ethernet, Wi-Fi, etc.) and virtual devices in the Guest VMs (also called "User OS"). Virtual Ethernet device could be shared by 1-2 Linux, Android, and Service OS guests for regular (i.e. non-AVB) traffic. All hypervisor para-virtualized I/O is implemented using the VirtIO specification Ethernet pass-through.
+
+Storage (eMMC)
+================
+ACRN hypervisor supported non-volatile R/W storage for the Service OS and Guest OS instances, supporting VM private storage and/or storage shared between Guest OS instances.
+
+USB (xDCI)
+================
+ACRN hypervisor supported assigning all USB xHCI and/or xDCI controllers to a Guest OS from the platform
+
+USB Mediator (xHCI and DRD)
+================
+ACRN hypervisor supported USB Mediator.
+
+CSME
+================
+ACRN hypervisor supported  CSME to a single Linux, Android, or RTOS guest or the Service OS even when in a virtualized environment.
+
+WiFi
+================
+ACRN hypervisor supported the passthrough assignment of the WiFi subsystem to the IVI, enabling control of the WiFI as an in-vehicle hotspot for 3rd party devices, providing 3rd party device applications access to the vehicle, and providing access of 3rd party devices to the TCU provided connectivity.
```

Comment:
change ``supported`` to ``supports``

spell ``WiFi`` consistently

change ``providing 3rd party`` to ``provides 3rd party``

change ``providing access`` to ``provides access``

keep the line length < 80 characters